parser/src/command/note.rs

```diff
@@ -0,0 +1,39 @@
+use crate::error::Error;
+use crate::token::{Token, Tokenizer};
+use std::fmt;
+
+#[derive(PartialEq, Eq, Debug)]
+pub enum NoteCommand {
+    Summary { title: String },
+}
+
+#[derive(PartialEq, Eq, Debug)]
+pub enum ParseError {
+    MissingTitle,
+}
+impl std::error::Error for ParseError {}
+impl fmt::Display for ParseError {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        match self {
+            ParseError::MissingTitle => write!(f, "missing required summary title"),
+        }
+    }
+}
+
+impl NoteCommand {
+    pub fn parse<'a>(input: &mut Tokenizer<'a>) -> Result<Option<Self>, Error<'a>> {
+        let mut toks = input.clone();
+        if let Some(Token::Word("note")) = toks.peek_token()? {
+            toks.next_token()?;
+            if let Some(Token::Word(title)) = toks.next_token()? {
```

Comment:
Let's support quoting the title as well -- i.e., Token::Quote -- which will let users supply a nicer name than just a single word.